guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -62,13 +74,13 @@ ifdef::katello[]
 # dnf module disable pulpcore
 ----
 endif::[]
-. Switch to the PostgreSQL 13 module:
+. On {EL} 8: Switch to the PostgreSQL 13 module:
```

Comment:
Pre-existing problem, but this part should be inside `ifdef::katello[]` since PostgreSQL isn't on a Smart Proxy with vanilla Foreman. Only with Katello.

Reply:
I moved the endif::[] to include this in the katello section.